cmake/modules/unittest.cmake

```diff
@@ -4,12 +4,23 @@ cmake_minimum_required(VERSION 3.20.0)
 
 enable_language(C CXX ASM)
 
+# Custom targets for compiler and linker flags.
+# These are normally created under FindTargetTools, but we can't include it here because unit
+# testing doesn't support cross compiling.
+add_custom_target(asm)
+add_custom_target(compiler)
+add_custom_target(compiler-cpp)
+add_custom_target(linker)
+
+include(FindHostTools)
```

Comment:
I propose to remove this:
```suggestion
```

Note, the line: `enable_language(C CXX ASM)` is moved further down.